plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/branch/BranchPresenter.java

```diff
@@ -361,16 +364,27 @@ protected void onSuccess(Branch result) {
 
```

Comment:
I think that amount of nesting levels should be reduced there.
6-7 nested blocks isn't good for reading code.